sdk/ai/azure-ai-generative/azure/ai/generative/synthetic/qa.py

```diff
@@ -193,7 +200,7 @@ def _modify_conversation_questions(self, questions) -> Tuple[List[str], Dict]:
 
     @distributed_trace
     @monitor_with_activity(logger, "QADataGenerator.Export", ActivityType.INTERNALCALL)
-    def export_to_file(self, output_path: str, qa_type: QAType, results: Union[List, List[List]]):
+    def export_to_file(self, output_path: str, qa_type: QAType, results: Union[List, List[List]], output_format: OutputStructure = OutputStructure.PROMPTFLOW, field_mapping: Dict[str,str] = {"chat_history_key": "chat_history", "question_key": "question", "answer_key": "answer"}):
```

Comment:
I dont think we need answer_key. we can always just keep at as 'ground_truth'

Reply:
Daniel listed answer_key in the bug as one of the things to allow to configure. I am good either way